core/src/display_object/movie_clip.rs

```diff
@@ -3248,8 +3248,10 @@ impl<'gc> TInteractiveObject<'gc> for MovieClip<'gc> {
                             Avm2MousePick::Miss
                         }
                     }
-                } else if child.as_interactive().is_none()
-                    && child.hit_test_shape(context, point, options)
+                } else if child
+                    .masker()
+                    .unwrap_or(child)
+                    .hit_test_shape(context, point, options)
```

Comment:
This needs to test both the mask _and_ the clip. If you modify the test to have the mask be 75px wide, click just to the right of the visible object and Flash says stage but Ruffle (this PR) says clip.